tap_recharge/streams.py

```diff
@@ -247,12 +247,13 @@ def get_records(
         url = None
 
         while paging:
-            records, links = self.client.get(path, url=url, params=self.params)
+            records, _ = self.client.get(path, url=url, params=self.params)
```

Comment:
The tap is not using a second return value of the `client.get()` function anywhere so remove it from `get()` function itself.

Reply:
Updated the **request** function to return in json response.